### DISCUSSION TITLE
Problem in the section "Extending Selection" of the tutorial?

### DISCUSSION THREAD

**Discussion by brabalan** [meow-edit/meow#570] [Q&A]
The description in the "Extending Selection" part of the tutorial is confusing. It says that after `meow-mark-word`, using `meow-next-word` "Moves forward to the end of the current word" and `meow-back-word` "Moves backward to the beginning of the current word". For me, `meow-next-word` extends the selection to include the word after the current selection, and `meow-back-word` extends the selection to include the word before the current selection, independently of the cursor position (i.e., `meow-reverse` changes nothing for theses commands). In particular, the only way to shorten the selection at this point in the tutorial is to run `meow-pop-selection'.

Is this a bug in the tutorial?

**Comment by DogLooksGood** [comment]
Here we have the [explanation](https://github.com/meow-edit/meow/blob/master/EXPLANATION.org#basic-concepts) for the design.

And yes, these are all the word related commands, and the `meow-pop-selection` is the only way how you shrink your selection with meow. But all builtin commands are still available, you can use `M-f` and `M-b` to do raw movement as well.

In most scenarios, we do accurate operations, we want to construct the selection in a most efficient way.  So we don't really need a method to shrink the selection, unless we made a mistake.  That's why we have this pop command as an undo command.

**Reply by brabalan** [reply]
Thank you for the link to the explanation, it does confirm what I observe and that the tutorial should be updated. Should I write a PR to this end?

**Reply by DogLooksGood** [reply]
We assume that the reader already knows the basics of emacs, so we won't go into that again.

**Reply by brabalan** [reply]
I’m not talking about the basics of emacs, but about the fact that word movement does not care about the selection direction. In the current tutorial, the text does not say it:
```
 Motions are useful for extending the current selection and for
 quick movement around the text. After selecting the word under
 the cursor with \\[meow-mark-word] you can extend the selection with some common
 movements listed below.

   \\[meow-next-word] - Moves forward to the end of the current word.
   \\[meow-back-word] - Moves backward to the beginning of the current word.
   \\[meow-next-symbol] - Moves to the end of the current symbol.
   \\[meow-back-symbol] - Moves to the start of the current symbol.
```

The problem is the meaning of "current". I would suggest something like:
```
   \\[meow-next-word] - Adds the next word to the selection.
   \\[meow-back-word] - Adds the previous word to the selection.
```

In addition, I would move the sentence
```
 Cursor position can be reversed with \\[meow-reverse] to extend the selection
 the other directions. 
```
to another part of the tutorial, since it does nothing for word extension.

**Reply by DogLooksGood** [reply]
You're right. Please submit a PR for this update!

**Reply by brabalan** [reply]
Done as #571
